diktat-rules/src/main/kotlin/org/cqfn/diktat/ruleset/rules/chapter4/NullChecksRule.kt

```diff
@@ -83,6 +92,27 @@ class NullChecksRule(configRules: List<RulesConfig>) : DiktatRule(
         }
     }
 
+    @Suppress("UnsafeCallOnNullableType")
+    private fun isFixable(condition: ASTNode,
+                          isEqualToNull: Boolean): Boolean {
+        // Handle cases with `break` word in blocks
+        val typePair = if (isEqualToNull) {
+            (ELSE to THEN)
+        } else {
+            (THEN to ELSE)
+        }
+        val isBlockInIfWithBreak = condition.getBreakNodeFromIf(typePair.first)
+        val isOneLineBlockInIfWithBreak = condition
+            .treeParent
+            .findChildByType(typePair.second)
+            ?.let { it.findChildByType(BLOCK) ?: it }
+            ?.let { astNode ->
+                astNode.hasChildOfType(BREAK) &&
+                        (astNode.psi as? KtBlockExpression)?.statements?.size != 1
+            } ?: false
+        return !(isBlockInIfWithBreak || isOneLineBlockInIfWithBreak)
```

Comment:
What about De Morgan's law?

```suggestion
        return (!isBlockInIfWithBreak && !isOneLineBlockInIfWithBreak)
```

Reply:
@petertrr?

Reply:
Yes, I agree, this should be changed